app/pages/patientdata/patientdata.less

```diff
@@ -33,16 +33,21 @@
 
 .patient-data-subnav-text {
   display: inline-block;
-  padding: @patient-data-subnav-vertical-padding @patient-data-subnav-horizontal-padding;
+  padding: @patient-data-subnav-vertical-padding 0 @patient-data-subnav-horizontal-padding 0;
```

Comment:
I definitely introduced that one! It's fixed in my latest commit. There was inconsistent padding on linked header items versus plain text. I've correct that and cleaned up a bit!